test/Microsoft.DotNet.Docker.Tests/ImageTests.cs

```diff
@@ -117,14 +113,14 @@ public void VerifyImages(ImageDescriptor imageDescriptor)
             {
                 CreateTestAppWithSdkImage(imageDescriptor, appSdkImage);
 
-                if (!imageDescriptor.IsArm)
+                if (!string.IsNullOrEmpty(imageDescriptor.SdkOsVariant))
                 {
                     VerifySdkImage_RunApp(imageDescriptor, appSdkImage);
                 }
 
                 VerifyRuntimeImage_FrameworkDependentApp(imageDescriptor, appSdkImage);
 
-                if (DockerHelper.IsLinuxContainerModeEnabled)
+                if (DockerHelper.IsLinuxContainerModeEnabled && !imageDescriptor.IsAlpine)
```

Comment:
Do you want to stick a comment here for the tracking bug for self-contained and alpine?

Reply:
Ah, I just read the comments - @janvorli was making one...

Reply:
If that dotnet/core-setup#3395 gets merged soon I will hold off on merging this PR.  If not I will add a comment.

Reply:
Merged.  I'll kick off an official build to get a new build out sooner.